src/wallet/db.h

```diff
@@ -122,11 +122,15 @@ class CWalletDBWrapper
     void IncrementUpdateCounter();
     unsigned int GetUpdateCounter();
 
+    std::atomic<unsigned int> nUpdateCounter;
+    unsigned int nLastSeen;
```

Comment:
In commit "Wallet: Replace pwalletMain with a vector of wallet pointers"

Should initialize these values.